TeamProject/GlobalWarmingGame/Interactions/Interactables/Buildings/IBuildable.cs

```diff
@@ -1,17 +1,14 @@
-﻿using GlobalWarmingGame.Action;
-using GlobalWarmingGame.ResourceItems;
+﻿using GlobalWarmingGame.ResourceItems;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
-namespace GlobalWarmingGame.Interactions.Interactables.Buildings
+namespace GlobalWarmingGame.Interactions.Interactables
```

Comment:
Might also need to change the location and namespace of this class
Should be GlobalWarmingGame/Interactions/Interactables/Buildings